ludwig/models/llm.py

```diff
@@ -167,9 +217,9 @@ def to_device(self, device):
             # if self.model.device != device:
             #     self.model = self.model.to(device)
 
-            self.eval_loss_metric = self.eval_loss_metric.to(device)
-            self.eval_additional_losses_metrics = self.eval_additional_losses_metrics.to(device)
-            self.output_features.update({k: v.to(device) for k, v in self.output_features.items()})
+            # self.eval_loss_metric = self.eval_loss_metric.to(device)
+            # self.eval_additional_losses_metrics = self.eval_additional_losses_metrics.to(device)
+            # self.output_features.update({k: v.to(device) for k, v in self.output_features.items()})
```

Comment:
Delete instead of uncomment? Also, is the reason we no longer need to do

```self.output_features.update({k: v.to(device) for k, v in self.output_features.items()})```

this is because we're not initializing these in the LLM model anymore?

Reply:
Yeah, they exist separately.